### DISCUSSION TITLE
After installing hacmini on NUC8BEH, I got stuck with half progress on the apple logo page

### DISCUSSION THREAD

**Discussion by jingzhongchangan** [osy86/HaC-Mini#1092] [Q&A] [answered]
My NUC was in Hackintosh but with some TB issues, so I tried to install HaC mini on the NUC, but after I patched it, I was unable to enter the system anymore.

The boot devices in the bios setting are:
![WechatIMG84](https://github.com/osy/HaC-Mini/assets/105959405/24436156-a93f-4ef8-ab46-797532233c8f)

The options on the opencore boot page are:
![WechatIMG83](https://github.com/osy/HaC-Mini/assets/105959405/e50cfc6e-5f49-4855-a2bb-48b9edf18e2f)

No matter whether I choose the "Install macOS Monterey" or the "macintosh hd" or others, all of them can't work properly. All are stuck on this page like this:
![WechatIMG85](https://github.com/osy/HaC-Mini/assets/105959405/afffb13d-8c27-448a-b09b-4b35c14f7eda)

Could anyone kindly guide me on how to undo the Hac Mini patcher or fix it?
Thanks in advance!

**Comment by immie-jay** [comment]
Reset NVRAM

**Reply by jingzhongchangan** [reply]
Which option should I enter after resetting NVRAM? I chose the Macintosh HD, and it still stuck there.

**Comment by xiaooloong** [comment, marked as answer]
这是冥王峡谷的项目，你的设备是豆子峡谷，他们是完全不同的两种硬件。
如果你指的是安装了 hacmini.pkg 之后无法启动，你可以将原本备份的 EFI 还原即可。
如果你指的是安装了 ThunderboltPatcher.app 后无法启动，你需要使用编程器为雷电主控刷回出厂固件。

**Reply by jingzhongchangan** [reply]
了解，感谢大佬！
